### PR TITLE
Modifications of GLM tools and add new plot tool

### PR DESCRIPTION
**Pull request by ColineRoyaux**
New options (Year as continuous in GLM, ...) in GLM tools and new tool to create time series ggplot from your GLM results

### CODE REVIEW

File: tools/PAMPA/pampa_macros.xml [hunk 1]
```diff
@@ -67,6 +69,11 @@
     <xml name="pampa_output_GLM">
         <data name="output_recap" from_work_dir="GLMSummaryFull.txt" format="txt" label="Simple statistics on chosen variables on ${on_string}"/>
         <data name="output_rate" from_work_dir="RatingGLM.txt" format="txt" label="Your analysis rating file on ${on_string}"/>
+        <collection type="list" name="output_GLM">
```
Comment (bgruening) [inline tools/PAMPA/pampa_macros.xml:72]: can we add a test for this outpu?

File: tools/PAMPA/PAMPA_PLOT_GLM.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        <param name="unitobstable" type="data" format="tabular" label="Unitobs table used for glm" help=""/>
+    </inputs>
+    <outputs>
+       <data name="report" from_work_dir="stdout.txt" label="Report" format="txt"/>
+       <collection type="list" name="plots">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+       </collection>
+    </outputs>
+    <tests>
+        <test>
```
Comment (bgruening) [inline tools/PAMPA/PAMPA_PLOT_GLM.xml:30]: can you add `expected_num_outputs` (https://docs.galaxyproject.org/en/latest/dev/schema.html#id78) here and maybe also in other tests?

**Review by bgruening — COMMENTED** [review]
Thanks @ColineRoyaux, this looks very nice! A few comments inside.

* we need to bump the tool versions of all tools
* @yvanlebras with this "special" shed.yml we probably need to update this file as well, correct?

File: tools/PAMPA/pampa_macros.xml (hunk 1 above)
Comment (ColineRoyaux) [reply tools/PAMPA/pampa_macros.xml:72]: It is an optional output hence I didn't used it for the test but I'll do it now ! Thank you !

File: tools/PAMPA/PAMPA_PLOT_GLM.xml (hunk 2 above)
Comment (ColineRoyaux) [reply tools/PAMPA/PAMPA_PLOT_GLM.xml:30]: Thank you very much ! I'll do that

**Comment by yvanlebras** [comment]
Version bumped! (If I am not saying a mistake)
Tank you so much Björn !!!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Wuhu! Cool beans!

**Comment by yvanlebras** [comment]
Wonder Coline "on the place"!!!!

**Comment by ColineRoyaux** [comment]
Thank you very much ! 😄

**Comment by yvanlebras** [comment]
Thank you Björn ! THANK YOU Coline !!!!!

**Pull request merged**